### PR TITLE
Ref #230 : Added a new Tag page and the tag generation logic.

### PR DESCRIPTION
**Pull request by Yann-BUTSCHER-EIRL**
Added :
- New tag controller
- New tag service
- New tag index template
- New tag PDF template

Modified :
- Base template : Added a link to the tag index page in the navbar

### CODE REVIEW

File: src/Controller/TagController.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?php
```
Comment (LucileDT) [inline src/Controller/TagController.php:1]: Please add comments to this controller to explain what it does

File: public/css/PDF/tag.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+@page {
+    margin: 0;
+    padding: 0;
+    width: 210mm;
+    max-width: 210mm;
+    height: 297mm;
+    max-height: 297mm;
+}
+
+.tag-row {
```
Comment (LucileDT) [inline public/css/PDF/tag.css:11]: No need for a class, you can use the `tr` selector

File: src/Service/TagService.php [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        string $projectDir
+    )
+    {
+        $this->twig = $twig;
+        $this->fileService = $fileService;
+        $this->projectDir = $projectDir;
+    }
+
+    /**
+     * Generate a single pdf file containing all the tags corresponding to a list of given people addresses.
```
Comment (LucileDT) [inline src/Service/TagService.php:50]: Could you add how many tags a page contains?

File: templates/PDF/Tag/_tag.html.twig [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+                        {% endif %}
+                    {% endfor %}
+                {% endif %}
+            {% endfor %}
+            {% if count != 0 %}
+            </tr>
+            {% endif %}
+        </table>
+    </body>
+</html>
```
Comment (LucileDT) [inline templates/PDF/Tag/_tag.html.twig:53]: Missing new line at end of file

File: templates/Tag/index.html.twig [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+                    href="{{ path('tag_generate') }}"
+                    class="btn btn-primary"
+                    title="{% trans %}Générer les étiquettes{% endtrans %}"
+                >
+                    <i class="icon ion-md-download"></i>&nbsp;Générer les étiquettes
+                </a>
+            </div>
+        </div>
+    </div>
+{% endblock %}
```
Comment (LucileDT) [inline templates/Tag/index.html.twig:33]: Missing new line at end of file

**Review by LucileDT — CHANGES_REQUESTED** [review]

**Comment by LucileDT** [comment]
@Yann-BUTSCHER-EIRL I need you to fix things related to my comments to merge your pull request.

**Review by LucileDT — APPROVED** [review]

**Comment by Yann-BUTSCHER-EIRL** [comment]
@LucileDT I added all the changes you requested.
Can you validate again and merge if all is good, please?

**Comment by LucileDT** [comment]
All good thanks!

**Pull request merged**